### PR TITLE
CodeCamp #34 [Docs] Add Chinese version of single_stage_as_rpn.md and test_results_submission.md

### PR DESCRIPTION
**Pull request by 137208**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by ZwwWayne** [comment]
Hi @137208 ,
Thanks for your kind PR. Could you recover to use the original pre-commit-config? This is necessary to keep the consistency and make your code meets the lint requirements.

File: docs/zh_cn/user_guides/single_stage_as_rpn.md [hunk 1]
```diff
@@ -1 +1,169 @@
...
+
+主要流程如下:
+
+1. 在Faster R-CNN 中使用`FCOS头`作为`RPN头`
+2. 评估候选区域
+3. 用预先训练的FCOS训练定制的Faster R-CNN
+
+## 在Faster R-CNN中使用`FCOS头`作为`RPN头`
+
+为了在Faster R-CNN中使用`FCOS头`作为`RPN头`，我们应该创建一个名为`configs/faster_rcnn/faster-rcnn_r50_fpn_fcos-rpn_1x_coco.py`的配置文件，并且在`configs/faster_rcnn/faster-rcnn_r50_fpn_fcos-rpn_1x_coco.py`中将' rpn_head '的设置替换为' bbox_head '的设置，此外我们仍然使用FCOS的瓶颈设置，步幅为`[8,16,32,64,128]`，并且更新' bbox_roi_extractor '的' featmap_stride '为' [8,16,32,64,128]`。为了避免损失变慢，我们在前1000次迭代而不是前500次迭代中应用预热，这意味着lr增长得更慢。相关配置如下:
```
Comment (ZwwWayne) [inline docs/zh_cn/user_guides/single_stage_as_rpn.md:15]: 为了在 Faster R-CNN 中
一般中英文混合的时候，英文前后会有空格与中文文字区分开，请按照这个要求统一修改全文

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,50 +1,10 @@
+# See https://pre-commit.com for more information
```
Comment (BIGWangYuDong) [inline .pre-commit-config.yaml:1]: unnecessary changes

File: docs/zh_cn/user_guides/test_results_submission.md [hunk 3]
```diff
@@ -1 +1,173 @@
-# 提交测试结果（待更新）
+# 提交测试结果
+
+## 全景分割测试结果提交
+
+下面几节介绍如何在 COCO 测试开发集上生成泛视分割模型的预测结果，并将预测提交到 [COCO评估服务器](https://competitions.codalab.org/competitions/19507)
+
+### 前提条件
+- 下载 [COCO测试数据集图像](http://images.cocodataset.org/zips/test2017.zip)，[测试图像信息](http://images.cocodataset.org/annotations/image_info_test2017.zip)，和[全景训练/相关注释](http://images.cocodataset.org/annotations/panoptic_annotations_trainval2017.zip)，然后解压缩它们，把 `test2017` 放到 ` data/coco/ ` ，把 json 文件和注释文件放到 `data/coco/annotations/` 。
```
Comment (zytx121) [inline docs/zh_cn/user_guides/test_results_submission.md:8]:
```suggestion
- 下载 [COCO测试数据集图像](http://images.cocodataset.org/zips/test2017.zip)，[测试图像信息](http://images.cocodataset.org/annotations/image_info_test2017.zip)，和[全景训练/相关注释](http://images.cocodataset.org/annotations/panoptic_annotations_trainval2017.zip)，然后解压缩它们，把 `test2017` 放到 ` data/coco/ `，把 json 文件和注释文件放到 `data/coco/annotations/` 。
```

File: docs/zh_cn/user_guides/single_stage_as_rpn.md [hunk 4]
```diff
@@ -1 +1,169 @@
...
+# testing with 8 GPUs
+bash
+tools/dist_test.sh
+configs/rpn/fcos-rpn_r50_fpn_1x_coco.py
+--work_dirs /faster-rcnn_r50_fpn_fcos-rpn_1x_coco/epoch_12.pth
+```
+
+## 用预先训练的 FCOS 训练定制的 Faster R-CNN
+
+预训练不仅加快了训练的收敛速度，而且提高了检测器的性能。因此，我们在这里给出一个例子来说明如何使用预先训练的 FCOS 作为 RPN 来加速训练和提高精度。假设我们想在 Faster R-CNN 中使用 `FCOSHead` 作为 rpn 头部，并与预先训练的人员一起训练 [`fcos_r50-caffe_fpn_gn-head_1x_coco`](https://download.openmmlab.com/mmdetection/v2.0/fcos/fcos_r50_caffe_fpn_gn-head_1x_coco/fcos_r50_caffe_fpn_gn-head_1x_coco-821213aa.pth)。 配置文件 `configs/faster_rcnn/faster-rcnn_r50-caffe_fpn_fcos- rpn_1x_copy .py` 的内容如下所示。注意， `fcos_r50-caffe_fpn_gn-head_1x_coco` 使用 ResNet50 的 caffe 版本，因此需要更新 `data_preprocessor` 中的像素平均值和 std。
```
Comment (zytx121) [inline docs/zh_cn/user_guides/single_stage_as_rpn.md:119]:
```suggestion
预训练不仅加快了训练的收敛速度，而且提高了检测器的性能。因此，我们在这里给出一个例子来说明如何使用预先训练的 FCOS 作为 RPN 来加速训练和提高精度。假设我们想在 Faster R-CNN 中使用 `FCOSHead` 作为 rpn 头部，并与预先训练的人员一起训练 [`fcos_r50-caffe_fpn_gn-head_1x_coco`](https://download.openmmlab.com/mmdetection/v2.0/fcos/fcos_r50_caffe_fpn_gn-head_1x_coco/fcos_r50_caffe_fpn_gn-head_1x_coco-821213aa.pth)。 配置文件 `configs/faster_rcnn/faster-rcnn_r50-caffe_fpn_fcos- rpn_1x_copy .py` 的内容如下所示。注意，`fcos_r50-caffe_fpn_gn-head_1x_coco` 使用 ResNet50 的 caffe 版本，因此需要更新 `data_preprocessor` 中的像素平均值和 std。
```

File: docs/zh_cn/user_guides/single_stage_as_rpn.md [hunk 5]
```diff
@@ -1 +1,169 @@
-# 将单阶段检测器作为 RPN（待更新）
+# 将单阶段检测器作为 RPN
+
+候选区域网络 (Region Proposal Network, RPN) 作为 [Faster R-CNN](https://arxiv.org/abs/1506.01497) 的一个子模块，将为 Faster R-CNN 的第二阶段产生候选区域。在 MMDetectio n里大多数的二阶段检测器使用 [`RPNHead`](../../../mmdet/models/dense_heads/rpn_head.py)作为候选区域网络来产生候选区域。然而，任何的单阶段检测器都可以作为候选区域网络，是因为他们对边界框的预测可以被视为是一种候选区域，并且因此能够在 R-CNN 中得到改进。因此在 MMDetection v3.0 中会支持将单阶段检测器作为 RPN 使用。
```
Comment (BIGWangYuDong) [inline docs/zh_cn/user_guides/single_stage_as_rpn.md:3]:
```suggestion
候选区域网络 (Region Proposal Network, RPN) 作为 [Faster R-CNN](https://arxiv.org/abs/1506.01497) 的一个子模块，将为 Faster R-CNN 的第二阶段产生候选区域。在 MMDetection 里大多数的二阶段检测器使用 [`RPNHead`](../../../mmdet/models/dense_heads/rpn_head.py) 作为候选区域网络来产生候选区域。然而，任何的单阶段检测器都可以作为候选区域网络，是因为他们对边界框的预测可以被视为是一种候选区域，并且因此能够在 R-CNN 中得到改进。因此在 MMDetection v3.0 中会支持将单阶段检测器作为 RPN 使用。
```

File: docs/zh_cn/user_guides/single_stage_as_rpn.md [hunk 6]
```diff
@@ -1 +1,169 @@
...
+# testing with 8 GPUs
+bash
+tools/dist_test.sh
+configs/rpn/fcos-rpn_r50_fpn_1x_coco.py
+--work_dirs /faster-rcnn_r50_fpn_fcos-rpn_1x_coco/epoch_12.pth
+```
+
+## 用预先训练的 FCOS 训练定制的 Faster R-CNN
+
+预训练不仅加快了训练的收敛速度，而且提高了检测器的性能。因此，我们在这里给出一个例子来说明如何使用预先训练的 FCOS 作为 RPN 来加速训练和提高精度。假设我们想在 Faster R-CNN 中使用 `FCOSHead` 作为 rpn 头部，并与预先训练的人员一起训练 [`fcos_r50-caffe_fpn_gn-head_1x_coco`](https://download.openmmlab.com/mmdetection/v2.0/fcos/fcos_r50_caffe_fpn_gn-head_1x_coco/fcos_r50_caffe_fpn_gn-head_1x_coco-821213aa.pth)。 配置文件 `configs/faster_rcnn/faster-rcnn_r50-caffe_fpn_fcos- rpn_1x_copy .py` 的内容如下所示。注意，`fcos_r50-caffe_fpn_gn-head_1x_coco` 使用 ResNet50 的 caffe 版本，因此需要更新 `data_preprocessor` 中的像素平均值和 std。
```
Comment (BIGWangYuDong) [inline docs/zh_cn/user_guides/single_stage_as_rpn.md:119]:
```suggestion
预训练不仅加快了训练的收敛速度，而且提高了检测器的性能。因此，我们在这里给出一个例子来说明如何使用预先训练的 FCOS 作为 RPN 来加速训练和提高精度。假设我们想在 Faster R-CNN 中使用 `FCOSHead` 作为 `rpn_head`，并加载预先训练权重来进行训练 [`fcos_r50-caffe_fpn_gn-head_1x_coco`](https://download.openmmlab.com/mmdetection/v2.0/fcos/fcos_r50_caffe_fpn_gn-head_1x_coco/fcos_r50_caffe_fpn_gn-head_1x_coco-821213aa.pth)。 配置文件 `configs/faster_rcnn/faster-rcnn_r50-caffe_fpn_fcos- rpn_1x_copy .py` 的内容如下所示。注意，`fcos_r50-caffe_fpn_gn-head_1x_coco` 使用 ResNet50 的 caffe 版本，因此需要更新 `data_preprocessor` 中的像素平均值和 std。
```

File: docs/zh_cn/user_guides/test_results_submission.md [hunk 7]
```diff
@@ -1 +1,174 @@
...
+```
+
+### 重命名文件并压缩结果
+
+推理之后，全景分割结果(一个 json 文件和一个存储掩码的目录)将在 `WORK_DIR` 中。我们应该按照 [COCO's Website](https://cocodataset.org/#upload)上的命名约定重新命名它们。最后，我们需要将 json 和存储掩码的目录压缩到 zip 文件中，并根据命名约定重命名该 zip 文件。注意， zip 文件应该**直接**包含上述两个文件。
+
+重命名文件和压缩结果的命令:
+
+```shell
+# In WORK_DIR, we have panoptic segmentation results: 'panoptic' and 'results.panoptic.json'.
```
Comment (RangiLyu) [inline docs/zh_cn/user_guides/test_results_submission.md:168]: 代码的注释也要翻译一下哦

**Review by RangiLyu — APPROVED** [review]
